QgisModelBaker/gui/panel/log_panel.py

```diff
@@ -52,6 +54,17 @@ def print_info(self, text, text_color=LogColor.COLOR_INFO):
         self.txtStdout.setTextColor(QColor(text_color))
         self.txtStdout.append(text)
 
+        if text_color == LogColor.COLOR_INFO:
```

Comment:
I'm a little bit unhappy about this because it looks a little strange to use the color as indicator of the type of message. It would be cooler - maybe - to have a type info / warning / fail / success... On the other hand, it's nice to be able to override the the color in case we have to visualize something specific (like we do it on topping).

So after all I would keep it like it is (no change requested). But maybe you have an idea to make it nicer...

Reply:
It was also my thought but did not came to a better idea in the moment

Reply:
Let's keep it dynamic :balloon:

Reply:
I took another look at it and I think it's a bit a "puff" with those colors, there is also another function in ili2dbutils called `color_log_text`. But maybe this is not used anymore?


> Dont concern these export.py and import_data.py and generate_project.py They should be removed in future...

Is that not possible at the moment? Or it just need to be done by someone?

Reply:
Needs to be done by someone. You can check it out if you have time.